dicoogle/src/main/java/pt/ua/dicoogle/server/web/webapp/WEB-INF/js/stores/searchStore.js

```diff
@@ -30,7 +30,7 @@ const SearchStore = Reflux.createStore({
           self._contents = data;
 
           //DEBUG WAIT
-          setTimeout(self.triggerWithDelay, 700)
+          setTimeout(self.triggerWithDelay, 50)
```

Comment:
Maybe we could clear the timeout entirely? The need for it might have emerged from other bad practices that we've most likely cleaned up by now.